webapp/.eslintrc.json

```diff
@@ -362,7 +361,14 @@
         "args": "after-used"
       }
     ],
-    "no-use-before-define": 0,
+    "no-use-before-define": [
+      2,
+      {
+        "classes": false,
+        "functions": false,
+        "variables": false
+      }
+    ],
```

Comment:
If this one was causing trouble, I think we should use the `@typescript-eslint` one instead for this

Reply:
i fixed it now, so i think we can let this rule be here.